src/__init__.py

```diff
@@ -174,31 +175,27 @@ def setup_mongodb(application):
                 f"{mongo_host}:27017/{mongo_db}"
             )
 
-    mongo_client = PyMongo(
+    mongo.init_app(
         application,
         uri=application.config["MONGO_URI"],
         uuidRepresentation="standard",
         serverSelectionTimeoutMS=2000,
         socketTimeoutMS=10000,
     )
-    application.db = mongo_client.db
 
     # Initialize collections and indices in case they don't exist already
     # Automatically expire jobs after 7 days if nothing runs them
-    application.db.jobs.create_index(
+    mongo.db.jobs.create_index(
```

Comment:
Is it okay that you don't remove the `db` field here and further?

Reply:
Yes, "mongo" here is a PyMongo object, which contains the db object.